### ISSUE TITLE
null is not an object (evaluating 's().exec(e)[0]')

### ISSUE THREAD

**Issue by ismael-gonzalez-sierra** [ci-dashboard/gitlab-ci-dashboard#3]
Hi, i have a new problem, I am checking with  v6.4.3-alpha.2  and v6.4.3-alpha.1.

I am using differents browser (Safari, Chrome and Firefox).

`Something went wrong. Make sure the configuration is ok and your Gitlab is up and running. Details: null is not an object (evaluating 's().exec(e)[0]')`

Regards,

**Comment by JosemaMP** [comment]
I have the same problem...

**Comment by emilianoeloi** [comment]
+ gitlab-ci-dashboard@6.4.4-alpha.1

**Comment by emilianoeloi** [comment]
I needed to reset the project repo and I'm having trouble publishing npm.

**Comment by emilianoeloi** [comment]
Hi, @ismael-gonzalez-sierra and @JosemaMP. Could you share the configuration/params that are passing! Be careful with your credentials, I made that mistake yesterday.
Thank you

**Comment by ismael-gonzalez-sierra** [comment]
Hi @emilianoeloi , you can see my config.json next:

```
{
  "dashboard": {
    "config": {
      "gitlab": "My gitlab-ce server",
      "token": "My personal token",
      "gitlabciProtocol": "https",
      "hideSuccessCards": false,
      "hideVersion": false,
      "interval": 60,
      "apiVersion": 4
    },
    "projects": [
      {
        "description": "My app iOS",
        "namespace": "IGS",
        "project": "My-App-iOS",
        "branch": "develop"
      }
    ]
  }
}
```

Regards :)

**Comment by sublimeye** [comment]
@ismael-gonzalez-sierra There is a bug / incomplete logic with tags sorting in `src/utils.js`. Basically if you have `tags` that are not `semver` compliant you'll get error. `semver-sort` can't parse a tag properly and fails. I'm going to make a pull request where this thing is fixed, not sure when, though

**Comment by marek-ganko** [comment]
In the meantime until my fix in PR #16 will be merged, you can use temporary published package with fixed for GitlabCI API v4: https://www.npmjs.com/package/gitlab-ci-v4-dashboard
